common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -376,7 +376,9 @@
             onContainerMouseEnter: function (event) {
                 event.preventDefault();
                 $(event.currentTarget).find('.lang').addClass('is-opened');
-                this.state.el.trigger('language_menu:show');
+                if ($(event.currentTarget).find('.lang').length) {
```

Comment:
Ahh, ok.  That makes sense.  It would be helpful to add a comment to that effect.

Wouldn't this fire an event if there is at least one language though, not strictly more than one?